hw/ip_templates/flash_ctrl/dv/flash_ctrl_base_sim_cfg.hjson

```diff
@@ -243,7 +243,8 @@
     {
       name: flash_ctrl_rw_serr
       uvm_test_seq: flash_ctrl_rw_vseq
-      run_opts: ["+scb_otf_en=1", "+ecc_mode=2", "+serr_pct=3"]
+      run_opts: ["+scb_otf_en=1", "+ecc_mode=2", "+serr_pct=3",
+                 "+otf_num_rw=100", "+otf_num_hr=1000"]
```

Comment:
We should probably check how this commit impacts the coverage of the block-level DV. I can trigger a run overnight if you would like me to.

Reply:
I would appreciate it.

Reply:
I ran a regression last night:

TOP_EARLGREY_BATCH_SIM Simulation Results (Summary)
### Wednesday July 24 2024 06:18:24 UTC
### GitHub Revision: [`e5c291b1f6`](https://github.com/lowrisc/opentitan/tree/e5c291b1f6ca4dbc6dda387acb3fa0a0e4875abf)
### Branch: flash_ctrl_read_sweep

|                                Name                                |  Passing  |  Total  |  Pass Rate  |  Coverage  |
|:------------------------------------------------------------------:|:---------:|:-------:|:-----------:|:----------:|
| [FLASH_CTRL](../../../ip_autogen/flash_ctrl/dv/latest/report.html) |   1274    |  1281   |   99.45 %   |  96.15 %   |

The coverage is slightly lower, could be due to statistics.

SCORE LINE COND TOGGLE FSM BRANCH ASSERT GROUP  
96.15 95.71 93.90 98.31 91.84 98.25 96.89 98.15  

Do you think we need to restore the _sim_cfg.hjson file to its former (and I think rather excessive) settings. Regardless, I think coverage needs to be analyzed to improve the tests.